lib/index.js

```diff
@@ -200,3 +189,25 @@ async function listenHttps() {
 async function listen() {
   await Promise.all([listenHttp(), listenHttps()]);
 }
+
+function _parseCerts({caFiles}) {
```

Comment:
See comment below about why this is to be renamed:

```suggestion
function _readCAFiles({caFiles}) {
```